### PR TITLE
Mani/dev 3683/masonry ref images

### PR DESCRIPTION
**Pull request by maniart**
Add support for masonry-style reference image layout inside search cards.
[Ticket Link](https://architizer.atlassian.net/browse/DEV-3683)

### CODE REVIEW

File: docs.md [hunk 1]
```diff
@@ -874,9 +874,9 @@ The examples below are the three main image-based card types on Source. They are
     </div>    
   </div>
 </div>
-<div class="row small-up-2 medium-up-3 large-up-4">
+<div class="row">
```
Comment (expandrew) [inline docs.md]: (Just adding this comment for the record, post-IRL-chat):

Check out the docs for [Foundation Block Grids](http://foundation.zurb.com/sites/docs/flex-grid.html#block-grids) - the card grid is showing kind of funny after this change

Comment (maniart) [inline docs.md]: Thanks @expandrew - Updated.

**Comment by maniart** [comment]
@expandrew  - Cool, Should we go ahead, merge & create a task for what we discussed?

**Comment by expandrew** [comment]
Yeah, go ahead and merge - we have a larger discussion to figure out how our docs site should work so I think we should have that separately.

**Pull request merged**